setup.cfg

```diff
@@ -0,0 +1,2 @@
+[aliases]
+test = pytest
```

Comment:
Please no.
I pretty happy with `pytest` name.

Reply:
It's an official way of integration + it worked this way before: https://github.com/aio-libs/multidict/pull/146/files/9e271b8286eda8237ad569495095c2b3793d81bc#diff-2eeaed663bd0d25b7e608891384b7298L104

Reply:
AFAIK you don't even use this

Reply:
I'd disable possibility for `python setup.py test` to invoke `unittest`

Reply:
Finally got it.